README.md

```diff
@@ -643,29 +635,21 @@ posthtml().process(html, { render: jsx(options) }).then((result) => result.html)
   <tbody>
 </table>
 
-## Contributing
-
-See [PostHTML Guidelines](https://github.com/posthtml/posthtml/tree/master/docs) and [CONTRIBUTING](CONTRIBUTING.md).
-
-## LICENSE
-
-[MIT](LICENSE)
-
 
 [npm]: https://img.shields.io/npm/v/posthtml.svg
 [npm-url]: https://npmjs.com/package/posthtml
 
 [deps]: https://david-dm.org/posthtml/posthtml.svg
 [deps-url]: https://david-dm.org/posthtml/posthtml
 
-[build]: https://travis-ci.org/posthtml/posthtml.svg?branch=master
-[build-url]: https://travis-ci.org/posthtml/posthtml?branch=master
+[tests]: https://travis-ci.org/p8osthtml/posthtml.svg
```

Comment:
Sry that was the old flanky keyboard, sometimes with extra eight :D, I fix it asap